posthog/tasks/test/test_status_report.py

```diff
@@ -45,6 +45,12 @@ def test_team_status_report_event_counts(self) -> None:
             self.assertEqual(team_report["persons_count_new_in_period"], 2)
             self.assertEqual(team_report["persons_count_active_in_period"], 1)
 
+            usage_summary = status_report(dry_run=True).get("usage_summary")
+            self.assertEqual(usage_summary["events_used_in_period"], 3)  # type: ignore
```

Comment:
I think it's worth adding events to a separate org and making sure these are properly accounted. In addition I would also add events on an _internal metrics_ org to make sure those are not counted. Think it's worth adding more strenuous tests as we'll use this for billing.

Not specifically related with the goal of this PR but I would also add an assertion/test for edge cases of events in the boundaries of periods, to make sure they're not double counted in two reports.

Reply:
Agreed, added some cases for events outside of the period as well as some on an internal metrics team